config/vlan.py

```diff
@@ -95,6 +95,12 @@ def del_vlan(db, vid):
     # We need to restart dhcp_relay service after dhcpv6_relay config change
     dhcp_relay_util.handle_restart_dhcp_relay_service()
 
+    keys = [ (k, v) for k, v in db.cfgdb.get_table('VLAN_MEMBER') if k == 'Vlan{}'.format(vid) ]
```

Comment:
is this change related to this feature?

Reply:
This is reverted in the updated PR https://github.com/sonic-net/sonic-utilities/pull/2881